docs/sdks/python/agents.mdx

```diff
@@ -1,137 +1,348 @@
 ---
-title: AI Agents with LLMs and Skills
-sidebarTitle: Agents
+title: How to Use Agents
+sidebarTitle: AI Agents
 ---
 
-## Description
+Agents enable conversation with data, including structured and unstructured data connected to MindsDB.
 
-With MindsDB, you can create and deploy AI agents that comprise AI models and customizable skills such as knowledge bases and text-to-SQL.
+## Deploy Agents
 
-<p align="center">
-  <img src="/assets/agent_diagram.png" />
-</p>
+Here is the syntax for creating an agent:
 
-AI agents comprise of skills, such as text2sql and knowledge_base, and a conversational model.
-
-* Skills provide data resources to an agent, enabling it to answer questions about available data. Learn more about [skills here](/sdks/python/agents_skills). Learn more about [knowledge bases here](/sdks/python/agents_knowledge_bases).
-
-* A conversational model (like OpenAI) from LangChain utilizes [tools as skills](https://python.langchain.com/docs/how_to/#tools) to respond to user input. Users can customize these models with their own prompts to fit their use cases.
+```python
+# prerequisite: get an existing project
+project = server.get_project()
```

Comment:
There isn't really a need to do this, because the `Server` object returned when `mindsdb_sdk.connect()` is called is a `Project` in itself.